packages/edit-site/src/components/sidebar-edit-mode/page-panels/page-status.js

```diff
@@ -193,10 +196,17 @@ export default function PageStatus( {
 									selected={ status }
 								/>
 								{ status !== 'private' && (
-									<BaseControl
-										id={ `edit-site-change-status__password` }
-										label={ __( 'Password' ) }
-									>
+									<fieldset className="edit-site-change-status__password-fieldset">
```

Comment:
This class name can be removed as well.

Reply:
Same as above: consistency with the form used in the post editor, see https://github.com/WordPress/gutenberg/blob/fix/use-secure-password-input-missing-label/packages/editor/src/components/post-visibility/index.js/#L77